.travis.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         - cat /tmp/test-integration/containerd.log
         - cat /tmp/test-cri/cri-containerd.log
         - cat /tmp/test-cri/containerd.log
-      go: 1.9.x
+      go: "1.9.x"
```

Comment:
Let also add a 1.10 version for this test?
We still need to support standalone mode for a while.

Reply:
done